specification/web/resource-manager/Microsoft.Web/stable/2020-09-01/ResourceProvider.json

```diff
@@ -0,0 +1,1398 @@
+        "responses": {
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "../2020-06-01/CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/validate": {
+      "post": {
+        "summary": "Validate if a resource can be created.",
+        "description": "Description for Validate if a resource can be created.",
+        "operationId": "Validate",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "validateRequest",
+            "in": "body",
+            "description": "Request with the resources to validate.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ValidateRequest"
+            }
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ValidateResponse"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "../2020-06-01/CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/validateMoveResources": {
+      "post": {
+        "summary": "Validate whether a resource can be moved.",
+        "description": "Description for Validate whether a resource can be moved.",
+        "operationId": "ValidateMove",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "moveResourceEnvelope",
+            "in": "body",
+            "description": "Object that represents the resource to move.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CsmMoveResourceEnvelope"
+            }
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "../2020-06-01/CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
```

Comment:
As of API version 2020-05-01, resources must begin adding the common systemData object property to the top-level property set on read responses (responses for PUT, GET, and PATCH).
RPC: https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc027-systemdata-support

You can reference [common type](https://github.com/Azure/azure-rest-api-specs/blob/20283a103ef5216f805fccc7ace6b6fc26c0cfbd/specification/common-types/resource-management/v2/types.json#L445) or have a copy of it.

Reply:
Added